frontend/src/concepts/pipelines/content/configurePipelinesServer/ConfigurePipelinesServerModal.tsx

```diff
@@ -64,7 +64,10 @@ export const ConfigurePipelinesServerModal: React.FC<ConfigurePipelinesServerMod
 
   const submit = () => {
     const objectStorage: PipelineServerConfigType['objectStorage'] = {
-      newValue: config.objectStorage.newValue,
+      newValue: config.objectStorage.newValue.map((entry) => ({
+        key: entry.key,
```

Comment:
would `{ ...entry, value: entry.value.trim() }` be good just to show nothing is being done except trimming the value?

Reply:
I like it, please make this change @DaoDaoNoCode